css-overflow-5/Overview.bs

```diff
@@ -885,51 +888,46 @@ Appendix B: Scroll navigation controls</h2>
 However, there is little feedback to the user regarding the current content being viewed, and
 the interaction model does not match the expectations of modern accessible UI components.
 
-This specification adds a mechanism for creating scroll navigation controls.
-The active marker, reflecting the current position, can be styled to give the user an indication of which section they are in.
-The set of markers are treated as a component following the accessibility guidelines for keyboard navigation within components.
+This specification adds a mechanism for creating scroll navigation controls within groups of <{a}> elements.
+A group is created through the <a href="https://open-ui.org/components/focusgroup.explainer/">focusgroup</a> attribute
+on an ancestor containing one or more <{a}> elements whose <a spec=html>indicated part</a> is on the current page.
+Within each group, the active marker reflects the current scroll position, and can be styled to give the user an indication of which section they are in.
 
 Use cases include a table of contents with links to relevant contents,
 markers for scrolling carousel pages,
 and scrollable tab panels.
 
 Issue: Add images representing these examples.
 
-Issue: Explore whether scrolltarget can be more directly associated with anchor tags.
-
-<h3 id="scroll-target"><{button/scrolltarget}> attribute</h3>
+<h3 id="scroll-target">{{HTMLLinkElement/scrollTargetElement}} attribute</h3>
 
-	The <dfn element-attr for=button>scrolltarget</dfn> attribute turns a <{button}> element into a [=scroll marker control=].
-	This takes the ID of the element to target as its value.
+	An <{a}> element has its {{HTMLLinkElement/scrollTargetElement}}
+	initialized to the <a spec=html>indicated part</a> of its <dfn element-attr for=a>href</dfn> attribute value if it is on the current page,
+	turning it into a [=scroll marker control=].
```

Comment:
An "indicated part" is something a document has (based on its url), not a url itself. Rephrase to something like:

> An a element, if it's a local link, has its scrollTargetElement initialized to the part of the [=document=] that would be the [=indicated part=] if the a element's url was the current url.

(Using whatever the correct version of terms like "local link" and "current url" are.)

Reply:
Looks like the [select the indicated part](https://html.spec.whatwg.org/multipage/browsing-the-web.html#select-the-indicated-part) algorithm already returns null if the target URL is not local, so I can invoke that.